Source/Core/Core/Config/GraphicsSettings.h

```diff
@@ -104,6 +105,26 @@ extern const Info<bool> GFX_ENHANCE_FORCE_TRUE_COLOR;
 extern const Info<bool> GFX_ENHANCE_DISABLE_COPY_FILTER;
 extern const Info<bool> GFX_ENHANCE_ARBITRARY_MIPMAP_DETECTION;
 extern const Info<float> GFX_ENHANCE_ARBITRARY_MIPMAP_DETECTION_THRESHOLD;
+extern const Info<bool> GFX_ENHANCE_HDR_OUTPUT;
+
+// Color.Correction
+
+static constexpr float GFX_CC_GAME_GAMMA_MIN = 2.2f;
```

Comment:
I don't think these should go in here because this whole file is for _settings_.  If these need to be source code wide I'd put them in `Constants.h`

Reply:
There's some similar instances for min/max constants above, that's why I put them there.

Reply:
Those min/max values are actual settings though, not constants.  Just felt a bit odd to me to have them in  here.  Will leave it up to someone else to decide, it's a small nit.